data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1701 @@
+				accept
+			label resistance
+			`A middle-aged Saryd, with little hair and a dense beard approaches you in the spaceport. You recognize him as Elirom, one of the Lunarium leaders you met on Remote Blue. "Talk in your ship, could we?" He whispers once he's close. "Here, listening they are..." You bring him to your ship, and he continues once the hatch closes. "Looking for our members all over <planet>, Heliarch ships are. Received word, I have, that due to <date>, many more Punishers are. If they get there before evacuated our members, we have, afraid I am that unfeasible to save them, it will become. Still waiting for a ship, <bunks> of them are, and get back there before the Punishers arrive on <date>, our other ships will not. Captain, help us, you must. Compensate you with 386,730 credits, I will."`
+			choice
+				`	"Of course, I'll head there right away."`
+					goto accept
+				`	"Can't they just lay low until the Punishers have gone away?"`
+			`	"No, no," he shakes his head. "Find them out, they would. They always do." His eyes wander around, as he takes a step back. He blinks repeatedly for a second, then looks back at you. "Seen what they do to the prisoners they take, I have, Captain. If there they remain, found they will be, and lost <bunks> more, I would have."`
+			label accept
+			`	Before you open up your hatch for him to leave, he adds, "Students and their teachers, they are. Picking them up from a school trip, after canceled on them, the original transport did, you are. Bring them back home to Second Viridian for their tests, you will. Asked to do so by the captain of the original transport ship, you were." He urges you to go there as soon as possible to pick them up, and leaves your ship.`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort with the <bunks> bunks for the students hasn't arrived in the system yet. Better depart and wait for it.`
+
+mission "Resistance: Evacuation 2"
+	name "Transport The Students"
+	description "Now that you've picked up the <bunks> students, bring them to their home planet of <destination>."
+	passengers 33
+	landing
+	source "Fourth Shadow"
+	destination "Second Viridian"
+	to offer
+		has "Resistance: Evacuation 1: done"
+		not "Heliarch Investigation 2: active"
+		not "Heliarch License: done"
+	to fail
+		has "Heliarch Investigation 2: active"
+	on offer
+		conversation
+			`A line of Kimek, with some Saryds and Arachi in their midst rush to your ship even before you've finished landing. It's the students, carrying very little luggage, and hastily coming inside once you open the hatch. Given the young appearance of all of them, you struggle to guess which ones are the teachers. After the last of them comes in, you look outside for a while, confused. Only <bunks> came in, not the 35 that Elirom mentioned.`
```

Comment:
Near the start he says this `Supposed to transport <bunks> students and teachers`.
It's not explicit there, since the `<bunks>` replacement is used instead of the number itself.
It's just the standard to use the replacement words when possible.